front50-core/src/main/groovy/com/netflix/spinnaker/front50/exception/InvalidApplicationRequestException.groovy

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.front50.exception
+
+import com.netflix.hystrix.exception.HystrixBadRequestException
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+public class InvalidApplicationRequestException extends HystrixBadRequestException {
```

Comment:
+1. 

Noticed a couple other exceptions in `Front50` that weren't using `@InheritConstructors` so maybe we can chase those down as well.